README.md

```diff
@@ -21,7 +21,25 @@ Currently the following variables are supported:
 
 ### General
 
-* `nfs_server_var_name` - var\_name description
+* `nfs_server_shares` - List of NFS shares to create in
```

Comment:
Heh, this might be the second time that the way the lines are wrapped here has confused a reviewer.  Check the rendered output of README.md and let me know if it still looks strange.

Reply:
OMG I didn't know that view existed. I've been looking for that feature and wondering why it wasn't there. This is awesome.